app/backend/gwells/database.py

```diff
@@ -19,21 +19,19 @@
 
 
 engines = {
-    'sqlite': 'django.db.backends.sqlite3',
     'postgresql': 'django.db.backends.postgresql',
+    'postgis': 'django.contrib.gis.db.backends.postgis',
```

Comment:
Realistically we're never going to switch DB again - so I would just drop the entire engines section.